src/table/index.js

```diff
@@ -372,6 +372,20 @@ class TableField extends Component {
   }
   forceReRenderTable = () => this.forceUpdate();
 
+  handleRowColorChange = (row, rowIndx) => {
+    let tableName = this.props.name;
```

Comment:
How do we know the table name? Do we need to pass an extra prop?

Reply:
Samip , we do not need to pass any extra prop, it is already coming